src/bim_parse.c

```diff
@@ -15,6 +15,8 @@
 #include <plinkio/bim.h>
 #include <plinkio/bim_parse.h>
 
+#include <./plinkio_private.h>
```

Comment:
```suggestion
#include "plinkio_private.h"
```

Reply:
I fixed it as suggested.

Reply:
Sorry, it wasn't fixed properly. I fixed it again. (https://github.com/mfranberg/libplinkio/pull/34/commits/7ae141ce816b549462cccee4281b672bceb29dc2)